pkg/api/pod/util.go

```diff
@@ -542,29 +520,21 @@ func overheadInUse(podSpec *api.PodSpec) bool {
 }
 
 // procMountInUse returns true if the pod spec is non-nil and has a SecurityContext's ProcMount field set to a non-default value
-func procMountInUse(podSpec *api.PodSpec) bool {
+func procMountInUse(podSpec *api.PodSpec) (inUse bool) {
```

Comment:
I'd probably prefer to still explicitly return a variable (`var inUse bool; ...; return inUse`).  Past experience / precedent within kubernetes has been that reviewers / readers struggle with naked returns.